contrib/azure/azure-coreos-cluster

```diff
@@ -85,6 +88,11 @@ if not args.ssh_cert and not args.ssh_thumb:
       args.ssh_cert = './ssh-cert.cer'
   print 'Generated SSH certificate with thumbprint ' + args.ssh_thumb
 
+# generate coreos discovery url
+if not args.no_discovery_url:
+  print 'Generating new CoreOS discovery URL for the cluster'
+  subprocess.call(['bash', '-c', 'python ./create-azure-user-data $(curl -s https://discovery.etcd.io/new)'])
```

Comment:
Does it make sense to use the fully-qualified path to `create-azure-user-data` here, so we can run this script from any directory?